crapssim/bet.py

```diff
@@ -253,38 +261,24 @@ def __init__(self, bet_amount: float):
 
 
 class Field(Bet):
-    """
-    Parameters
-    ----------
-    double : list
-        Set of numbers that pay double on the field bet (default = [2,12])
-    triple : list
-        Set of numbers that pay triple on the field bet (default = [])
-    """
-
-    def __init__(self, bet_amount: float, double: list[int] = [2, 12], triple: list[int] = []):
-        super().__init__(bet_amount)
+    def __init__(self, bet_amount: float):
+        super().__init__(bet_amount, None)
         self.name: str = "Field"
-        self.double_winning_numbers: list[int] = double
-        self.triple_winning_numbers: list[int] = triple
         self.winning_numbers: list[int] = [2, 3, 4, 9, 10, 11, 12]
         self.losing_numbers: list[int] = [5, 6, 7, 8]
 
-    def _update_bet(self, table_object: "Table", dice_object: Dice) -> tuple[str | None, float, bool]:
-        status: str | None = None
-        win_amount: float = 0
+    @property
+    def payout_ratio(self):
+        return self.table.payouts['field_ratios'][self.table.dice.total]

```

Comment:
Should there be some logic where the `payout_ratio` is always defined?  If `table.dice.total` is in `table.payouts['field_ratios']` then `Field(1).payout_ratio` returns a value, but otherwise it gives an error.  I'm not sure if it's better to raise an error when there shouldn't be a payout, or to have some default value of 0.0.  Thoughts?

Regardless, it would be good to add some testing here, especially when table.payouts['field_ratios'] changes.  I've started a test that might work (Note: 2 of them fail based on the above comment)

```python
@pytest.mark.parametrize('dice_total, correct_payout_ratio', [
    (2, 2.0),
    (3, 1.0),
    (7, 0.0),
    (0, 0.0)
])
def test_field_payout_ratio(dice_total, correct_payout_ratio):
    table = Table()
    player = Player(100)
    bet = Field(1)
    bet.player = player
    bet.table = table
    table.dice.total = dice_total

    assert bet.payout_ratio == correct_payout_ratio
```